src/components/Thumbnail/Thumbnail.tsx

```diff
@@ -3,18 +3,30 @@ import { Button } from "./Button";
 import { Description } from "./Description";
 import { Image } from "./Image";
 import { Name } from "./Name";
+import { Date } from "./Date";
 
 interface IProps {
+  date?: string;
   description: string;
   name: string;
   icon: string;
   alt: string;
+  button: string;
 }
-export const Thumbnail: FC<IProps> = ({ description, name, icon, alt }) => (
-  <div className="p-8 w-full shadow" style={{ height: "304px" }}>
-    <Image icon={icon} alt={alt} classname="mb-2" />
-    <Name>{name}</Name>
-    <Description className="mb-16">{description}</Description>
-    <Button>Zistiť viac</Button>
+
+export const Thumbnail: FC<IProps> = ({
+  description,
+  name,
+  icon,
+  alt,
+  date,
+  button
+}) => (
+  <div className="p-8 shadow h-full flex-col flex justify-between font-lato">
+    <Image icon={icon} alt={alt} classname="mb-8" />
+    <Date>{date}</Date>
```

Comment:
I think we need this {date && <Date>{date}</Date>}

Reply:
before changes it was {date && <Date>{date}</Date>} 
<img width="370" alt="why" src="https://user-images.githubusercontent.com/32729870/58820777-c5b5f080-8633-11e9-9a3f-29ca07bc26c7.png">